src/appkit.cs

```diff
@@ -9307,6 +9329,12 @@ partial interface NSImage : NSCopying, NSSecureCoding, NSPasteboardReading, NSPa
 
 		[Export ("layerContentsForContentsScale:")]
 		NSObject GetLayerContentsForContentsScale (nfloat layerContentsScale);
+
+		[Mac (10,16)]
+		[Static]
+		[Export ("imageWithSystemSymbolName:accessibilityDescription:")]
+		[return: NullAllowed]
+		NSImage GetSystemSymbol (string symbolName, [NullAllowed] string accessibilityDescription);
```

Comment:
`GetSystemSymbolImage` since we're returning an `NSImage`?

Reply:
We are inside NSImage already so it's NSImage.GetSystemSymbol()